### PR TITLE
Stats Menu adjustments and made it so Skill Tree menu is toggleable

### PR DESCRIPTION
**Pull request by uchennaponuigbo**
Made Stats menu compact and nicer. Next PR will be essentially a rewrite of the skill tree work

### CODE REVIEW

File: the-lost-penguin/Assets/Scripts/PlayerStats.cs [hunk 1]
```diff
@@ -62,10 +63,10 @@ void Update()
         //both enemy and item scripts should have an exp field to be used
 
         //testing, remove when done
-        // if(Input.GetKeyDown(KeyCode.J))
-        // {
-        //     AddExp(1);
-        // }
+        if(Input.GetKey(KeyCode.J))
+        {
+            AddExp(1);
```
Comment (Ghop11) [inline the-lost-penguin/Assets/Scripts/PlayerStats.cs]: What is this for? Should the player be able to spam "J" and just get more exp?

**Review by Ghop11 — CHANGES_REQUESTED** [review]
I left one comment for you.

File: the-lost-penguin/Assets/Scripts/PlayerStats.cs (hunk 1 above)
Comment (uchennaponuigbo) [reply the-lost-penguin/Assets/Scripts/PlayerStats.cs]: debugging purposes. I forgot to comment it out

**Review by Ghop11 — APPROVED** [review]
lgtm

**Pull request merged**